website/static/js/profile.js

```diff
@@ -587,9 +636,36 @@ var ListViewModel = function(ContentModel, urls, modes) {
         }
         return true;
     });
+
+    self.missingRequiredfield = ko.computed(function() {
+        for (var i=0; i<self.contents().length; i++) {
+            if (self.jobsOrSchools() === JOBS) {
```

Comment:
See previous comment re: encapsulation--the base `ListViewModel` should not have to know about `JOBS` and `SCHOOLS`.